exercises/03_key_value_header.py

```diff
@@ -36,18 +36,28 @@
 			info_data.append(line.strip().split())	
 
 #printing dictionary and list to make sure values are in place
+print("Header Data")
 print(hdr_data)
+print("\nColumn Names")
 print(column_names)
 
-
 #creating a pandas dataframe from the data in the text file
 df = pd.DataFrame(info_data)
 df.columns = column_names
+print("\nData Frame")
 print(df)
 
+# columns in df object are of Series type, following lines convert it and i0 columns to ndarray with float type values
+it_data = np.array(df.it, dtype="float")
+i0_data = np.array(df.i0, dtype="float")
+
 #plotting
+plt.ion()
 plt.figure(0)
 plt.clf()
-
-np.log(df.it/df.i0)
-
+plt.plot(np.log(it_data/i0_data), color='#be0119')  #scarlet color

```

Comment:
great! for a final step, can you plot versus energy?
xlabel should then be "energy (keV)"

Reply:
would it be like this?
'''python
energy_data = np.array(df.energy, dtype="float")

plt.figure(1)
plt.clf()
plt.plot( ((np.log(it_data/i0_data)) , energy_data))
plt.show()
'''

Reply:
(use the back tick symbol not the quote for code blocks)

almost, `energy_data` should be first

Reply:
I'll give it a try, with the back tick symbol
```python
plt.plot(energy_data, np.log(it_data/i0_data), color='#be0119')  #scarlet color
```

Reply:
It worked, thanks !